ykrt/src/mt.rs

```diff
@@ -357,10 +252,19 @@ impl MT {
             let hl = unsafe { ls.hot_location() };
             let hl_ptr = hl as *mut _ as *mut ();
             match hl {
-                HotLocation::Compiled(tr) => {
+                HotLocation::Compiled(compiled_trace) => {
```

Comment:
I know you don't like short variable names, but that's the convention in this file, so let's not mix two coding styles.

Reply:
the reason I did this is because there are so many different kinds of traces involved. In this file I think it's important to differentiate from an IR trace and a compiled trace.

Reply:
`ctr` if you wish. But let's not mash styles like this.

Reply:
I went with `tr`, as `ctr` invokes "counter" in my mind. See 0df447df6.